company/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+# Create your models here.
+class Company(models.Model):
+    INDUSTRY_CHOICES=[
+        ('entertainment', 'Entertainment'),
+        ('manufacturing', 'Manufactiring'),
+        ('real_estate', 'Real_estate'),
+        ('accommodation services', 'Accommodation services'),
+        ('financial and insurance', 'Financial and Insurance'),
+        ('information and communication', 'Information and Communication'),
+    ]
+    companyName = models.CharField(max_length=60)
```

Comment:
In python we don't use camel case, change these and everywhere else to company_name and be consistent with that

Reply:
okay i'll work on it